runtime/nls/j9ti/jvmti.nls

```diff
@@ -638,3 +638,17 @@ J9NLS_JVMTI_COM_SUN_HOTSPOT_EVENTS_VIRTUAL_THREAD_UNMOUNT.explanation=Internatio
 J9NLS_JVMTI_COM_SUN_HOTSPOT_EVENTS_VIRTUAL_THREAD_UNMOUNT.system_action=None
 J9NLS_JVMTI_COM_SUN_HOTSPOT_EVENTS_VIRTUAL_THREAD_UNMOUNT.user_response=None
 # END NON-TRANSLATABLE
+
+J9NLS_J9JVMTI_EVENT_OPENJ9_VM_CHECKPOINT=At VM pre-checkpoint phase.
+# START NON-TRANSLATABLE
+J9NLS_J9JVMTI_EVENT_OPENJ9_VM_CHECKPOINT.explanation=Internationalized description of a JVMTI extension
+J9NLS_J9JVMTI_EVENT_OPENJ9_VM_CHECKPOINT.system_action=None
+J9NLS_J9JVMTI_EVENT_OPENJ9_VM_CHECKPOINT.user_response=None
+# END NON-TRANSLATABLE
+
+J9NLS_J9JVMTI_EVENT_OPENJ9_VM_RESTORE=At VM post-restore phase.
+# START NON-TRANSLATABLE
+J9NLS_J9JVMTI_EVENT_OPENJ9_VM_RESTORE.explanation=Internationalized description of a JVMTI extension
```

Comment:
"Internationalized description of a JVMTI extension" doesn't seem like appropriate (English) explanations.

Reply:
Fixed in https://github.com/eclipse-openj9/openj9/commit/5768c727965bfb39506877857277d72b3e11d107